analysisstore/client/commands.py

```diff
@@ -10,10 +10,9 @@
 
 
 class AnalysisClient:
-    """Your path to AnalysisStore(yiiiiissssss, camelcase"""
+    """Client used to pass messages between analysisstore server and apps"""
     def __init__(self, host='localhost:8999'):
```

Comment:
We already have a standard method (at least for my services). This was my attempt to try something new and a little more refined. I am glad it was able to start some discussion. 
All in all, if I deviate from the standard and find out the new method is superior, I will be switching all configurations to the same method. RIght now, everything but this one follows the same standard